apps/cloud/src/api/see_food.ts

```diff
@@ -0,0 +1,31 @@
+import Store from '@raincatcher/store';
+import webapi, { ApiService, StoreApiService, WebApiConfig } from '@raincatcher/webapi';
+import * as express from 'express';
+
+interface SeeFood {
+  id: string;
+  isHotdog: boolean;
+  mikeHuntScore: number;
+}
+
+const config: WebApiConfig = {
+  limits: true
```

Comment:
Another follow up idea is to make config optional. Maybe we can extend it by for example specifying what CRUD operations are possible?